tools/topology/sof/pipe-kfbm-capture.m4

```diff
@@ -43,6 +44,25 @@ C_CONTROLBYTES(KPB, PIPELINE_ID,
 	,
 	KPB_priv)
 
+# Volume Mixer control with max value of 32
+C_CONTROLMIXER(KWD Capture Volume, PIPELINE_ID,
+	CONTROLMIXER_OPS(volsw, 256 binds the mixer control to volume get/put handlers, 256, 256),
+	CONTROLMIXER_MAX(, 32),
+	false,
+	CONTROLMIXER_TLV(TLV 32 steps from -64dB to 0dB for 2dB, vtlv_m64s2),
```

Comment:
You should use for capture vtlv_m50s1 with 80 steps. It provides the volume boost that's often required with microphones.

Reply:
good point,  changing it, thanks Seppo.